clickhouse/datadog_checks/clickhouse/clickhouse.py

```diff
@@ -76,11 +76,23 @@ def validate_config(self):
         if not self._server:
             raise ConfigurationError('the `server` setting is required')
 
+    def ping_clickhouse(self):
+        return self._client.connection.ping()
+
     def connect(self):
         if self._client is not None:
-            # Already connected.
-            self.service_check(self.SERVICE_CHECK_CONNECT, self.OK, tags=self._tags)
-            return
+            self.log.debug('Clickhouse client already exists. Pinging Clickhouse Server.')
+            try:
+                if self.ping_clickhouse():
+                    self.service_check(self.SERVICE_CHECK_CONNECT, self.OK, tags=self._tags)
```

Comment:
This makes the submission on line [125](https://github.com/DataDog/integrations-core/pull/7006/files#diff-54d2bcfaacb57a3325f40103c6c2a08fR125) redundant